### ISSUE TITLE
Consider to add `spring-integration-test` into test dependencies when we poll Spring Integration

### ISSUE THREAD

**Issue by artembilan** [spring-io/initializr#1001]
The `spring-integration-test` brings many useful testing features for integration solutions: https://docs.spring.io/spring-integration/reference/html/#testing

So, would be great to have it included into `test` scope dependencies when we select Spring Integration from start.spring.io

**Comment by dsyer** [comment]
...or Spring Cloud Stream?

**Comment by artembilan** [comment]
Yes. that's OK, too. There are a lot in Spring Cloud Stream based on Spring Integration. 
So, having SI testing utils in classpath should be a benefit as well.
